lib-utilities/config/config_test.go

```diff
@@ -131,7 +131,25 @@ func TestSetConfiguration(t *testing.T) {
                 "Fabrics",
                 "Managers"
 		],
-		"SupportedPluginTypes" : ["Compute", "Fabric"]
+		"SupportedPluginTypes" : ["Compute", "Fabric"],
+		"ConnectionMethodConf":[
+			{
+				"ConnectionMethodType":"Redfish",
+				"ConnectionMethodVariant":"Compute:iLO_v1.0.0"
+			},
+			{
+				"ConnectionMethodType":"Redfish",
+				"ConnectionMethodVariant":"Compute:GRF_v1.0.0"
+			},
+			{
+				"ConnectionMethodType":"Redfish",
+				"ConnectionMethodVariant":"Fabric:CFM_v1.0.0"
+			},
+			{
+				"ConnectionMethodType":"Redfish",
+				"ConnectionMethodVariant":"Storage:STG_v1.0.0"
```

Comment:
Storage is missing in SupportPluginTypes.

Reply:
added